spec/lib/common/client/middleware/request/multipart_request.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
```

Comment:
The multipart middleware that faraday provides requires the IO objects to be wrapped in FaradayIOUpload object, this sets it up properly. But because there can also be a message payload, I have to set any message payload as an IOStream then make it a FaradayIOUpload object with mime type of application/json